pyqtgraph/functions.py

```diff
@@ -119,35 +119,58 @@ def siFormat(x, precision=3, suffix='', space=True, error=None, minVal=1e-25, al
 
 def siParse(s, regex=FLOAT_REGEX, suffix=None):
     """Convert a value written in SI notation to a tuple (number, si_prefix, suffix).
-    
+
     Example::
-    
-        siParse('100 μV")  # returns ('100', 'μ', 'V')
+
+        siParse('100 µV")  # returns ('100', 'µ', 'V')
+
+    Note that in the above example, the µ symbol is the "micro sign" (UTF-8
+    0xC2B5), as opposed to the Greek letter mu (UTF-8 0xCEBC).
```

Comment:
nice note, I did not know this!

Reply:
Funny you mention that - I didn't either until figuring out why example that was already in the docstring wasn't working. Hex editor was vital on this one!

Reply:
I can see why that can be useful for easier classification/consistency, but these essentially identical-looking (homoiconic?) UTF symbol are really annoying at times. See also different kinds of spaces, or also invisible spaces.